Source/Csla.Shared/Rules/IAuthorizationContext.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
```

Comment:
This file needs the standard copyright header

Reply:
Should be fixed now.